src/util.rs

```diff
@@ -137,6 +137,33 @@ pub fn respace(s: &str) -> String {
     s.split_whitespace().collect::<Vec<_>>().join(" ")
 }
 
+pub fn escape_brackets(s: &str) -> String {
+    s.split('[')
+        .fold("".to_string(), |acc, x| {
+            if acc == "" {
+                x.to_string()
+            } else {
+                if acc.ends_with("\\") {
+                    acc.to_owned() + "[" + &x.to_string()
+                } else {
+                    acc.to_owned() + "\\[" + &x.to_string()
+                }
+            }
+        })
+        .split(']')
+        .fold("".to_string(), |acc, x| {
+            if acc == "" {
+                x.to_string()
+            } else {
+                if acc.ends_with("\\") {
+                    acc.to_owned() + "[" + &x.to_string()
```

Comment:
Typo, I think `[` is meant to be `]`.

I see regressions like
```
-#[doc = "Word 2 \\[95:64\\]"]
+#[doc = "Word 2 \\[95:64\\["]
```

Reply:
Thanks for catching this. Do you want to do a PR or shall I?

Reply:
I can do the PR, tested it locally so the tiny changes are there